pkg/networkservice/common/heal/client.go

```diff
@@ -31,21 +32,26 @@ import (
 )
 
 type healClient struct {
-	chainCtx        context.Context
-	livelinessCheck LivelinessCheck
+	chainCtx              context.Context
+	livelinessCheck       LivelinessCheck
+	attemptAfter          time.Duration
+	livenessCheckInterval time.Duration
 }
 
 // NewClient - returns a new heal client chain element
 func NewClient(chainCtx context.Context, opts ...Option) networkservice.NetworkServiceClient {
-	o := &option{
-		livelinessCheck: func(_ *networkservice.Connection) bool { return false },
+	o := &options{
+		attemptAfter:          time.Millisecond * 200,
+		livenessCheckInterval: time.Millisecond * 500,
```

Comment:
Out of curiosity, why these default values?

Reply:
* `attemptAfter` is the same as before this PR.

* About `livenessCheckInterval`. In the latest revision of the PR, this value was removed; the user should provide its own timer instead in the `LivenessChecker` implementation.

  In the previous version of this PR (the one that you commented), `livenessCheckInterval` had roughly the same order of magnitude as the default `ping` utility interval. Also, it was tied to [these values](https://github.com/Chunosov/cmd-nsc/blob/6fca644fa41fe5a8d533375c088a1b010d8b7ee6/main.go#L74).

Reply:
@xzfc Are we using the 'ping' command, or are we sending the icmp echo messages directly from Go?

Reply:
We're sending ICMP messages directly, see https://github.com/networkservicemesh/sdk-kernel/pull/447.

Reply:
@xzfc Woot!